### ISSUE TITLE
Builder fields are not editable in a multi-language setup

### ISSUE THREAD

**Issue by sen-zo** [TimOetting/kirby-builder#54]
Builder fields are not editable anymore in multi-language setups (probably since Kirby version 2.5.3). At least in non-default languages.

There is an exception in builder/controller.php:75

`$fieldsetStructure = $this->fieldsetStructure($entry->_fieldset);`

$entry is `false` and therefore access to `_fieldset` throws an exception. It looks like something with the `$entryId` parameter is wrong.

**Steps to reproduce:**

1. Download the Kirby-LanguageKit: [https://github.com/getkirby/langkit](https://github.com/getkirby/langkit)
2. Add the builder-field
3. Add a field to the home.yml blueprint:
```
  builder:
    label: Sections
    type: builder
    fieldsets:
      bodytext:
        label: Body Text
        snippet: sections/bodytext
        fields:
          text:
            label: text
            type: textarea
```
4. Login to the panel and create a "bodytext" field in the page "home". The selected language should be the default language (en). Add some text, click "add" and "save".
5. Switch to the german version and click "edit" on the previously created field. Now, the previously mentioned exception is thrown.

After you click "save" in the german version, you will be able to edit the field. But if you go back to the english version, edit it and save, the german version will be broken again.

(This is on a Mac with MAMP Version 4.2.1)

Thanks for the nice plugin ;)

**Comment by sen-zo** [comment]
This might be caused by / is related to: [https://github.com/getkirby/panel/issues/1123](https://github.com/getkirby/panel/issues/1123)

**Comment by Galothus** [comment]
I tried this with langkit 2.5.8 on Win 10. 
After step 4 I don't get the exception. 
Pressing edit button on DE page does nothing. Pressing delete button on DE page results in Error The item could not be found.

Also the text that was written inside is not shown on both languages ( I wrote "lorem lorem")
![textnotshown](https://user-images.githubusercontent.com/17845708/34007856-f3d3eab8-e10b-11e7-8bc5-3c704e9fa8bd.png)
